assets/css/copy-button.css

```diff
@@ -14,7 +14,9 @@ pre .copy-button:focus {
   top: 5px;
   right: 5px;
   padding: 4px;
-  background-color: var(--codeBackground);
+  background-color: transparent;
+  backdrop-filter: blur(2px);
```

Comment:
```suggestion
  background-color: var(--codeBackground);
```

I tried it in a couple places but I found the blurred colors confusing. I did like the border though, can we revert it partially?

Reply:
understandle, i increased the contrast. if you still find it confusing i'll revert the change

before
![image](https://github.com/user-attachments/assets/7ea75cd8-fd2f-4d62-a00f-5eab527da868)

after
![image](https://github.com/user-attachments/assets/bf60bec7-5c7f-43de-bb1d-e23cdf11b307)